ExampleApps/SwiftUIExample/SwiftUIExampleTests/TestConstant.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  TestConstant.swift
+//  SwiftUIExampleTests
+//
+//  Created by Richard Gist on 7/20/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import Foundation
+
+enum TestConstant {
+    static var timeout: TimeInterval {
+        10.0 // Pipeline
+//        return 0.1; #warning("Local timeout should not be checked in")
```

Comment:
10s timeout?

Reply:
I didn't like that either.  The good news is that this 10 second timeout only applies in this context when ViewInspector is unable to call `.inspect` and the places where we are testing for `onFinish` or `onAbandon` to be called.  Both of these scenarios will only hit the 10 seconds if they completely fail.

Reply:
In my investigation of the pipeline, we could prune it down to 5 seconds and still be pretty safe.  3 seconds would be mostly safe, as I didn't see any test run all the way up to 3 seconds, but 1 test did get close.

Reply:
A heavier lift option would be to split the waiting for expectations into 2 lines.  There would be a long timeout wait **just** for the `inspect` call, and a shorter timeout on the `onFinish`.  If we wait for `inspect` then wait for `onFinish` then you would have up to 10 seconds to wait on inspecting (which should always pass) and then a shorter .5 second timeout that onFinish was called (which will probably already be called at this point).

Drawbacks to this plan: If you do the order wrong the test will become flakey again, Quite a bit of code change and will probably be forgotten in future tests. The longest wait will be 10.5 seconds for if everything catastrophically fails.

Positives: Passing tests will be unnoticeably longer, Failing tests for when something breaks in `onLoad` or `onAbandon` will happen quickly (expected to be .5 + time it took to inspect).

Reply:
I'd like to at least try to get it down if possible considering there appear to be quite a few places that I saw using that `timeout`. If there's nothing we can really do about it other than wait out the inspect call then it is what it is, and I can accept that.

Reply:
We can prune it down to 5 seconds at least.  @morganzellers has already suggested that change in his review so I'll take that.